system/libbase58/base58.c

```diff
@@ -38,7 +40,7 @@ bool b58tobin(void *bin, size_t *binszp, const char *b58, size_t b58sz)
     const unsigned char *b58u = (const unsigned char *)b58;
     unsigned char *binu = (unsigned char *)bin;
     size_t outisz = (binsz + 3) / 4;
-    uint32_t outi[outisz];
+    uint32_t *outi = (uint32_t*)alloca(outisz);
```

Comment:
This is not equivalent.  It needs outisz * sizeof(uint32_t).  Why the tests didn't catch it?

Reply:
This code is unused by us, so my tests left it out, I will fix and add tests.